src/main/java/com/hms_networks/americas/sc/extensions/system/application/SCAppRebootWatchdog.java

```diff
@@ -190,4 +221,25 @@ private static JSONObject getDefaultRebootWatchdogObject() throws JSONException
 
     return defaultRebootWatchdogObject;
   }
+
+  /**
+   * Sets the time the watchdog will accept servicing
+   *
+   * @param watchdogWaitTimeMillis the amount of time the watchdog should wait before accepting
+   *     servicing
+   */
+  private static void setWatchdogReadyTime(long watchdogWaitTimeMillis) {
+    watchdogReadyTimeMillis = System.currentTimeMillis() + watchdogWaitTimeMillis;
+  }
+
+  /**
+   * Checks if the watchdog has been running for a sufficient amount of time
+   *
+   * @return A boolean indicating if the watchdog has been running for a sufficient amount of time
+   */
+  private static boolean checkWatchdogWaitTimer() {
+    long currentTimeMillis = System.currentTimeMillis();
+
+    return (currentTimeMillis >= watchdogReadyTimeMillis) ? true : false;

```

Comment:
```suggestion
    return currentTimeMillis >= watchdogReadyTimeMillis;
```

`? true : false;` is redundant.

Reply:
fixed.

Reply:
I agree. I only caught this because IntelliJ shows a warning when you use `? true : false` so it is engraved in my mind always to check.